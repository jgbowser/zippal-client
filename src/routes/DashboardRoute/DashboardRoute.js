import React, {Component} from 'react'
import Dashboard from '../../components/Dashboard/Dashboard'


export default class DashboardRoute extends Component {

  
  render(){
    return (
      <section>
        <div>
          <h2> welcome!</h2>
          <Dashboard/>
        </div>
      </section>

    )
  }
}