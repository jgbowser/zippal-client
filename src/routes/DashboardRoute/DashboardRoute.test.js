import React from 'react'
import ReactDOM from 'react-dom'
import DashboardRoute from './DashboardRoute'

describe('DeshboardRoute Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<DashboardRoute />, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})