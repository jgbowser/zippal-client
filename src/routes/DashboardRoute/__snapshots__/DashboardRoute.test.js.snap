// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardRoute Component takes a snapshot of the DashboardRoute component 1`] = `
<section>
  <section
    className="dashboard"
  >
    <div
      className="welcome_text"
    >
      Welcome, 
      !
      <br />
       Write a message to your pals!
    </div>
    
    
    <section
      className="Active_Conversations"
    >
      <button
        className="Dashboard__newPal_button"
        onClick={[Function]}
      >
        Find a new Pal
      </button>
      <button
        className="Dashboard__newPal_button"
        onClick={[Function]}
      >
        Find a new Pal
      </button>
      <button
        className="Dashboard__newPal_button"
        onClick={[Function]}
      >
        Find a new Pal
      </button>
      <button
        className="Dashboard__newPal_button"
        onClick={[Function]}
      >
        Find a new Pal
      </button>
      <button
        className="Dashboard__newPal_button"
        onClick={[Function]}
      >
        Find a new Pal
      </button>
    </section>
  </section>
</section>
`;
